decidim-conferences/app/commands/decidim/conferences/admin/create_partner.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Conferences
+    module Admin
+      # A command with all the business logic when creating a new partner
+      # in the system.
+      class CreatePartner < Rectify::Command
+        # Public: Initializes the command.
+        #
+        # form - A form object with the params.
+        # conference - The Conference that will hold the speaker
+        def initialize(form, current_user, conference)
+          @form = form
+          @current_user = current_user
+          @conference = conference
```

Comment:
Shouldn't the conference be part of the form too?

Reply:
I think it's cleaner (you don't have to merge attributes from the form) but maybe I didn't review it correctly in other PRs.

Reply:
I still think that it is better to continue with the same system, for the whole decidim. Not in different ways according to the command...

Reply:
I agree to what @isaacmg410 says! 😄